### ISSUE TITLE
request for video demonstration of paper

### ISSUE THREAD

**Issue by AvAkanksh** [MJITG/PyTorch-HITNet-Hierarchical-Iterative-Tile-Refinement-Network-for-Real-time-Stereo-Matching#9]
Hello Mjitg, I see that you have implemented the code according to the paper by making different submodules, but because we don't have any video demonstration of how the architecture is working we are unable to understand it, so it would be great if you could upload a video demonstrating the algorithm, it need not cover all the part, could just explain the model and loss function, that will be more than enough. 

Thank you

Akanksh
